examples/complete/main.tf

```diff
@@ -74,7 +94,26 @@ module "mongodb" {
   resource_group_id   = module.resource_group.resource_group_id
```

Comment:
Need to add a dependency. You won't see the failure in the test but if you manually apply you will probably hit it.
``` hcl
  // Policy needs to be applied before CRB is created or there will be a failure
  depends_on = [ibm_iam_authorization_policy.policy]
```

Reply:
Actually, just ran it manually and it passed.

Reply:
You are right not needed